pkg/e2e/verify/namespace_webhook.go

```diff
@@ -159,6 +170,80 @@ var _ = ginkgo.Describe(namespaceWebhookTestName, func() {
 				Expect(err).NotTo(HaveOccurred())
 			}
 		}, viper.GetFloat64(config.Tests.PollingTimeout))
+
+		util.GinkgoIt("dedicated admin can not create PrometheusRule", func() {
+			for privilegedNamespace := range PRIVILEGED_NAMESPACES {
+				prome2etestrule := e2eCreatePrometheusRule(prometheusName, privilegedNamespace, DUMMY_USER, "dedicated-admins", h)
+				err := wait.PollImmediate(5*time.Second, 1*time.Minute, func() (bool, error) {
+					_, err := h.Prometheusop().MonitoringV1().PrometheusRules(privilegedNamespace).Create(context.TODO(), prome2etestrule, metav1.CreateOptions{})
+					if err != nil {
```

Comment:
Do you know specifically what type of error you expect this to generate? This test may be best to explicitly check for that type of error. One of the helper functions in `k8s.io/apimachinery/pkg/api/errors` may then be able to help. 

ie. I wonder if it would be a `Forbidden` type of error? In which case:
```suggestion
					if err != nil && errors.IsForbidden(err) { 
```

Reply:
The same will apply for the other places in the tests where we expect an RBAC failure to occur, ie the one right below this one.